src/components/lib/cos_kernel_api.c

```diff
@@ -109,7 +110,7 @@ __mem_bump_alloc(struct cos_compinfo *__ci, int km)
 		/* are we dealing with a kernel memory allocation? */
 		syscall_op_t op = km ? CAPTBL_OP_MEM_RETYPE2KERN : CAPTBL_OP_MEM_RETYPE2USER;
 
-		if (call_cap_op(ci->pgtbl_cap, op, ret, 0, 0, 0)) return 0;
+		if (call_cap_op(ci->local_mi.pgtbl_cap, op, ret, 0, 0, 0)) return 0;
```

Comment:
@ryuxin you're right but if we name it `UNTYPED_PT`, I was thinking, once it's retyped, if it would still be in that PT or would be mapped into SELF_PT. I understand that it is mainly physical memory mappings that will be part of UNTYPED_PT, virtual mappings will be in SELF_PT.